tests/unit/src/test/scala/tests/DefinitionLspSuite.scala

```diff
@@ -121,7 +121,7 @@ class DefinitionLspSuite
            |import org.scalatest.funsuite.AnyFunSuite/*AnyFunSuite.scala*/
            |object MainSuite/*L6*/ extends AnyFunSuite/*AnyFunSuite.scala*/ {
            |  test/*AnyFunSuiteLike.scala*/(testName/*<no symbol>*/) {
-           |    val condition/*L8*/ = Main/*Main.scala:5*/.message/*Main.scala:4*/.contains/*String.java*/("Hello")
+           |    val condition/*L8*/ = Main/*Main.scala:5*/.message/*<no symbol>*/.contains/*String.java*/("Hello")
```

Comment:
This change is expected since there is no actual `message` anymore, it was changed and the code no longer parses. We would guess it previously based on outdated information.